src/spec/utils.spec.coffee

```diff
@@ -52,3 +53,10 @@ describe 'Utils', ->
     ], (o) ->
       it "should get time in milliseconds for '#{o.type}'", ->
         expect(Utils.getTime(o.amount, o.type)).toBe o.expected_time
+
+  describe ':: getVersion', ->
+
+    it "should return the current version of the sphere node sdk", ->
+      pjson = require appRoot + '/package.json'
+      expect(Utils.getVersion()).toBe pjson.version
+      expect(Utils.getVersion()).toMatch(/^(\d+\.)?(\d+\.)?(\*|\d+)$/)
```

Comment:
According to npm semver `0.0.0-1` is a valid version. But this regex will fail here. Maybe it's better to add [semver](https://docs.npmjs.com/misc/semver) as a dependency and use it as a validator?

Reply:
ok I amended this - semver is now a dev dependency